.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - master
+
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
should we be running CI on more operating systems or is this unlikely to fail on Mac & Windows?

Reply:
There shouldn't be any platform-specific code that would fail in this package. It doesn't interact with terminals, filesystems, networks, etc, so we should be okay.

If that turns out not to be true in the future, we can revisit this and expand what platforms we check on!